### ISSUE TITLE
dataset and parameter for PBMC dataset

### ISSUE THREAD

**Issue by mrbeann** [TingheZhang/T-GEM#1]
Thanks for the great work! But it seems there only have bulk results. Can you also release the PBMC results?

**Comment by TingheZhang** [comment]
Hi, Thanks for being interesting at our work. 

I have uploaded the PBMC code label and data. 

You simply run the code via commend : 
python G-TEM_PBMC_1l.py --head_num 5 --learning_rate 0.0001 --act_fun leakyrelu --batch_size 16 --epoch 1 --do_val --dropout_rate 0.3 --result_dir model_res/1l/ --model_dir model/1l/

**Comment by mrbeann** [comment]
Thank you for your support. I'll try it.
